lib/screens/friends/suggestion_friends/suggestion_friends_widget.dart

```diff
@@ -0,0 +1,176 @@
+            ),
+          ),
+          Padding(
+            padding: EdgeInsets.only(right: 16, left: 24),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Row(
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.only(top: 14.0),
+                      child: SvgPicture.asset(
+                          'assets/icons/suggestion-friend.svg'),
+                    ),
+                    SizedBox(width: 8),
+                    Padding(
+                      padding: const EdgeInsets.only(top: 14.0),
+                      child: RichText(
+                          text: TextSpan(children: [
+                        TextSpan(
+                            text:
+                                AppLocalizations.of(context)!.friendSuggestions,
+                            style: TextStyle(
+                              fontSize: 16,
+                              color: LightColors.grey,
+                              fontWeight: FontWeight.w400,
+                            )),
+                      ])),
+                    ),
+                  ],
+                ),
+                TextButton(
+                  onPressed: () {
+                    Future.delayed(Duration(milliseconds: 100), () {
+                      controller.insertPage(AddFriends(
+                        displayModal: true,
+                      ));
+                    });
+                  },
+                  child: Row(
+                    children: [
+                      Text(
+                        AppLocalizations.of(context)!.seeAll,
+                        style: TextStyle(
+                          fontWeight: FontWeight.w400,
+                          fontSize: 14,
+                          color: LightColors.blue,
+                        ),
+                      ),
+                      SizedBox(width: 4),
+                      Icon(
+                        Icons.arrow_forward_ios,
+                        color: LightColors.blue,
+                        size: 14,
+                      )
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+          SingleChildScrollView(
+            scrollDirection: Axis.horizontal,
+            child: Row(
+              children: [
+                buttonToSuggestionFriends(),
+                Row(
+                  children: widget.friendsStore.suggestionFriends.friends!
+                      .map((e) => ItemFriend(friend: e!))
+                      .toList(),
+                ),
+              ],
+            ),
+          )
+        ],
+      ),
+    );
+  }
+
+  Widget buttonToSuggestionFriends() {
+    return GestureDetector(
+      onTap: () {
+        Future.delayed(Duration(milliseconds: 100), () {
```

Comment:
Por que tem esse future.delayed?

Reply:
sei lá, como eram os mesmos widgets de add friends eu só copiei colei e alterei o que era preciso

Reply:
Remove se não houver necessidade, e retira também de onde você pegou se for o caso.

Reply:
rapaz, quero mexer na tela de friends não
tá uma porra aquilo